pkg/logger/logger.go

```
@@ -18,26 +21,65 @@
+	LogLevelOff:     5,
 }
 
 type Logger struct {
-	LogLevel LogLevel
-	File     string
+	LogLevel    LogLevel
+	File        string
+	AtmosLogger *log.Logger
 }
 
 func NewLogger(logLevel LogLevel, file string) (*Logger, error) {
```

golangci-lint (1.62.2)

[error] cyclomatic: function NewLogger has cyclomatic complexity 11 (> max enabled 10)

(revive)